### DISCUSSION TITLE
GitHub Suspended My Other Account with No Explanation

### DISCUSSION THREAD

**Discussion by hyper-text-coffee-pot** [community/community#122615] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

I've waited patiently for a month now waiting on a response from GitHub on why my other account was suspended. I opened a ticket like your help docs recommend and I've been waiting for even the slightest indication as to what went on with my other account. It's really unfortunate and I've been trying my best not to be too frustrated over the ordeal, but after doing some digging in that time I've found this same situation to be the fate of many other GitHub users. As such I've stopped using GitHub altogether as I'm not willing to risk basing my own repositories off of a service who can just shut the doors without warning.

This is an egregious overreach of control over people's repositories for a service claiming to be a primary promoter of the open-source community. Had I not had my own local copies of my repos, of which contained months worth of work, you will have effectively stolen my code from me. I wouldn't be quite so frustrated had you simply sent me an email with reasons and a specific detail on which term or condition your system seems to think I violated. And then had been willing to reach a resolution for my so-called violation. This is a respectable process.

I sincerely hope this issue and methodology of auto-flagging and suspending accounts without any warning or information on the alleged infraction is not the norm and that I'm maybe somehow in the small minority of folks unfortunate enough to experience this. Even so, I was a loyal GitHub Copilot fanatic and was singing its praises, paying a monthly sub for the service and after this whole scenario I've cancelled that service, found alternatives and tell people to be careful working with you guys. Really, again, unfortunate that this is the way of GitHub since you used to be a super fan favorite amongst devs.

On the off-chance that this makes its way to someone who actually seems to care about my concerns here is my ticket number:
2715192

Sincerely,

A disappointed user.

**Comment by luoabner** [comment]
I have the same problem. But I don't know why. GitHub ticket seems to be useless.  #122270

**Reply by hyper-text-coffee-pot** [reply]
If you're interested in living vicariously through my frustration feel free to read my blog post. As it's basically been the only sense of control over the situation I could come up with.

TL;DR; I am now using https://codeium.com/ and https://about.gitlab.com/ for all of my stuff. I've abandoned all projects through GitHub.

Here is my blog post with some research and tidbits I found from others across the web:
https://blog.hypertextcoffeepot.com/the-rise-and-fall-of-github

Cheers

**Comment by hyper-text-coffee-pot** [comment]
Here's the response I received from GitHub's support team on my other account. It tells me absolutely nothing about what they're claiming I did to have my account suspended and they completely shut the door on me to being willing to sort out the issue. This cannot seriously be any way to run a business these days. Unreal. All I want is for a support rep or literally anyone at this company to act like they care one shred about their customers. I'm clearly not a bot, and I'm clearly not attempting to do anything actively against their terms and conditions. All I was trying to do was set up a GitHub Action to help deploy my project to a VM hosted at another company. That's about 90% of why anyone would ever use Actions. I do not understand why they are giving me the run around:

"Hello,
 
Thank you for writing in.
 
The account has restrictions imposed because it appears to have been taking part in activity which goes against our Terms of Service:
 
[Terms of Service](https://docs.github.com/github/site-policy/github-terms-of-service)
 
In particular, any repositories that use GitHub Actions solely to interact with 3rd party websites, to engage in incentivized activities, or for general computing purposes may fall afoul of the GitHub Additional Product Terms:
 
[GitHub Additional Product Terms](https://docs.github.com/site-policy/github-terms/github-terms-for-additional-products-and-features#actions)
 
For this reason, we will not be removing the restrictions from this account.
 
Best regards,
GitHub Support"

**Reply by hyper-text-coffee-pot** [reply]
And given that GitHub is not a hosting company and I would never run VMs with them if they had them, if the claim is that trying to deploy my project to a 3rd party, being DigitalOcean, then I don't even believe that GitHub Actions are a valid solution for anyone.

**Comment by DTtempo** [comment]
Experiencing the same thing here, its been almost 60 days since I opened my ticket and no response from the support team at all. Not even to the support ticket which was opened by my employers Github Pro account which supposedly should have a 24 hour response time.

Someone posted an AI generated reply to my thread and as a result the community manager locked my entire thread, is that reasonable?

https://github.com/orgs/community/discussions/122123

**Reply by hyper-text-coffee-pot** [reply]
As much as it stinks for anyone else to have to go through this, I am somewhat relieved it's not just me. The reality is that they are making me, and others, feel like garbage in having to experience this. Think if this was how the rest of the major online services worked and every company treated customers this way. Pretty absurd.
